heatmap/src/heatmap.rs

```diff
@@ -2,36 +2,27 @@
 // Licensed under the Apache License, Version 2.0
 // http://www.apache.org/licenses/LICENSE-2.0
 
-use crate::Heatmap;
-use crate::HeatmapError;
-use core::sync::atomic::AtomicU64;
-use core::sync::atomic::AtomicUsize;
+use crate::Error;
+use crate::*;
+use core::sync::atomic::*;
 
-use rustcommon_atomics::{Atomic, Ordering};
-use rustcommon_histogram::{AtomicCounter, AtomicHistogram, Counter, Indexing};
-use rustcommon_time::*;
+use histogram::{Bucket, Histogram};
 
 /// AtomicHeatmaps are concurrent datastructures which store counts for
 /// timestamped values over a configured time range with individual histograms
 /// arranged in a ring buffer. Increments occur in the most recent slice in the
 /// buffer, unless they are newer than that slice may hold. When this happens,
 /// old slices are cleared and reused. This configuration results in a fully
 /// pre-allocated datastructure with concurrent read-write access.
```

Comment:
This comment should be updated to talk about `Heatmap` now instead of `AtomicHeatmap`.

Also, I suggest having a short intro line and then putting a more detailed description into a separate paragraph since it'll look better in rustdoc.